js/script.js

```diff
@@ -0,0 +1,14 @@
+const navlinks = document.querySelector('.navlinks');
+const menu = document.querySelector('.fa-bars');
+const close = document.querySelector('.fa-x');
+// console.log(menu)
+
+menu.addEventListener('click', () => {
+  navlinks.classList.add('open');
+  close.classList.add('show');
```

Comment:
- [ ] You did well  😃at implementing some changes by the previous reviewer, but as stated by the previous reviewer the open mobile menu functionality is working, but when I click on any option the mobile menu not working, so please try to fix that and make sure When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.💯

> Check below to see the difference 😊⤵️


| My mobile menu | Your mobile menu| 
| --- | --- |
|    ![Recording 2023-02-10 at 15 50 45](https://user-images.githubusercontent.com/83360636/218108243-99894073-5d88-407e-922c-dfaf9c2d4b87.gif)  |    ![Recording 2023-02-10 at 15 49 30](https://user-images.githubusercontent.com/83360636/218107973-7c28f9dc-7d69-46e3-9671-1ff820330f85.gif)   |